packages/path_provider/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.2.0
+
+* Android: `getApplicationSupportDirectory` enabled by using `getFilesDir`.
```

Comment:
```suggestion
* On Android, `getApplicationSupportDirectory` is now supported and uses `getFilesDir`.
* `getStorageDirectory` now returns null instead of throwing an exception if no external files directory is available.
```